README.md

```diff
@@ -68,12 +68,15 @@ which HIDAPI supports.  Since it relies on a 3rd party library, building it
 is optional but recommended because it is so useful when debugging hardware.
 
 ## What Does the API Look Like?
-The API provides the the most commonly used HID functions including sending
-and receiving of input, output, and feature reports.  The sample program,
+The API provides the most commonly used HID functions including sending
+and receiving of input, output, and feature reports. The sample program,
 which communicates with a heavily hacked up version of the Microchip USB
 Generic HID sample looks like this (with error checking removed for
 simplicity):
 
+`Warning: Only run the code you understand, only against devices which firmware 
+you control. Writing data at random to your HID devices can break them.`
```

Comment:
I would use *s instead of `s, so to make it bold because it looks a bit weird in this preformatted style.

Reply:
done